domains/julianwe17.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Julianwe17",
+           "email": "julian.in.hbr@gmail.com",
+           "discord": "1055740356764901437"
+        },
+    
+        "record": {
+            "TXT": "julianwe17"
```

Comment:
TXT records cannot be used with CNAME records. CNAME records must be standalone records.